### PR TITLE
Set PS4 in jobs via global defaults

### PR DESCRIPTION
**Pull request by JanZerebecki**
DO NOT MERGE, WIP

### CODE REVIEW

**Comment by JanZerebecki** [comment]
Suggested in: https://github.com/SUSE-Cloud/automation/pull/2764#issuecomment-422788171

File: jenkins/ci.suse.de/000-global-defaults.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+- defaults:
+    name: global
+    parameters:
+      - choice:
+          name: PS4
+          choices:
+            - "+${BASH_SOURCE/$HOME/~}@${LINENO}(${FUNCNAME[0]}): "
```
Comment (aspiers) [inline jenkins/ci.suse.de/000-global-defaults.yaml]: Based on [the Travis failure](https://travis-ci.org/SUSE-Cloud/automation/builds/430799232) I suspect you're missing a level of escaping here, but I'm not sure.

Comment (JanZerebecki) [reply jenkins/ci.suse.de/000-global-defaults.yaml]: Yes in JJB values that can have parameters any curly braces need to be escaped by doubling them.

Comment (aspiers) [reply jenkins/ci.suse.de/000-global-defaults.yaml]: OK, so are you gonna fix this? ;-)

Comment (JanZerebecki) [reply jenkins/ci.suse.de/000-global-defaults.yaml]: Done.

**Comment by JanZerebecki** [comment]
Due to discontinuation and this touching every job I'll close this.

**Pull request closed without merging**